### ISSUE TITLE
Typo in the train/run.lua

### ISSUE THREAD

**Issue by SilviaLauraPintea** [vbalnt/pnnet#4]
Hi,

There seems to be a typo in the training of the network ("train/run.lua") at line 35:

_model1:add(nn.Linear(64 \* 8 \* 8, 256))_

the 256 should be replaced by 128, as the model described outputs 128 dimensions.

_model1:add(nn.Linear(64 \* 8 \* 8, 128))_

All the best & great work!

**Comment by vbalnt** [comment]
Hi, thanks for the issue report. 

In the paper we describe two possible dimensionalities for the output, since the 256-dim features perform slightly better in matching tasks. 

You can either use 256 or 128 in the final Linear layer.
